docs/cheatsheet.rst

```diff
@@ -33,6 +33,7 @@ Global Variables
 - ``block.chainid`` (``uint``): current chain id
 - ``block.coinbase`` (``address payable``): current block miner's address
 - ``block.difficulty`` (``uint``): current block difficulty
+- ``block.prevrandao`` (``uint``): random number provided by the beacon chain
```

Comment:
I think having both of them listed without any comment would indicate both are *semantically* available at the same time.

Reply:
Please, see the last changes here: https://github.com/ethereum/solidity/pull/13759/commits/ea9f1846d8023149b727261ea71cac07a1cb86cb